src/contract_wrappers/registries/security_token_registry_wrapper.ts

```diff
@@ -726,6 +746,10 @@ export default class SecurityTokenRegistryWrapper extends ContractWrapper {
     assert.assert(params.ticker.length > 0 && params.ticker.length <= 10, 'Bad ticker');
     assert.assert(params.deployedAt.getTime() > new Date(0).getTime(), 'Bad deployed date');
     assert.isNonZeroETHAddressHex('owner', params.owner);
+    assert.isNonZeroETHAddressHex(
```

Comment:
Shouldn't the assertions for this function be exactly the same as the ones for `modifySecurityToken`?

If that is the case, then you should consider refactoring those assertions into a common function